DMCompiler/DM/Expressions/Procs.cs

```diff
@@ -133,5 +133,14 @@ class ProcCall : DMExpression {
                 proc.Call(procRef);
             }
         }
+
+        public override bool TryAsJsonRepresentation(out object json) {
+            json = null;
+            if (!DMCompiler.Settings.SuppressUnimplementedWarnings)
+            {
+                DMCompiler.Warning(new CompilerWarning(Location, $"DMM overrides for expression {GetType()} are not implemented"));
+            }
+            return true; //TODO
+        }
```

Comment:
Proc calls cannot be serialized into json. Was this added because of a `rgb()` call or similar?